memgpt/server/rest_api/interface.py

```diff
@@ -355,13 +357,24 @@ def get_generator(self) -> AsyncGenerator:
             raise StopIteration("The stream has not been started or has been ended.")
         return self._create_generator()
 
-    def _push_to_buffer(self, item: Union[MemGPTMessage, LegacyMemGPTMessage, MessageStreamStatus]):
+    def _push_to_buffer(
+        self,
+        item: Union[
+            # signal on SSE stream status [DONE_GEN], [DONE_STEP], [DONE]
+            MessageStreamStatus,
+            # the non-streaming message types
+            MemGPTMessage,
+            LegacyMemGPTMessage,
+            # the streaming message types
```

Comment:
@sarahwooders the "streaming message types" actually includes `MemGPTMessage`, since `MemGPTMessage` types (`FunctionCall`, `InternalMonologue`, ...) all natively support streaming (as mentioned in an earlier comment).